components/layout/fragment_tree/fragment.rs

```diff
@@ -168,14 +168,19 @@ impl Fragment {
                 let fragment = fragment.borrow();
                 fragment.offset_by_containing_block(&fragment.scrollable_overflow())
             },
-            _ => self.scrollable_overflow_for_parent(),
+            _ => self.scrollable_overflow_for_parent(AxesOverflow::default()),
         }
     }
 
-    pub(crate) fn scrollable_overflow_for_parent(&self) -> PhysicalRect<Au> {
+    pub(crate) fn scrollable_overflow_for_parent(
+        &self,
+        overflow_style: AxesOverflow,
```

Comment:
```suggestion
        parent_overflow_style: AxesOverflow,
```
nit: this will be easier to read